ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixParticipant.java

```diff
@@ -87,7 +89,9 @@ public void participate(List<AmbryHealthReport> ambryHealthReports) throws IOExc
     stateMachineEngine.registerStateModelFactory(LeaderStandbySMD.name, new AmbryStateModelFactory());
     registerHealthReportTasks(stateMachineEngine, ambryHealthReports);
     try {
+      helixAdmin.close();
       manager.connect();
+      helixAdmin = manager.getClusterManagmentTool();
```

Comment:
Add a comment as to why we close and reassign this way?

Reply:
How about noting that the reason `helixAdmin` was initialized in the constructor before connecting and announcing (participating) is so that the caller can do any administrative operations in Helix before participating?